server/src/main/kotlin/suwayomi/tachidesk/graphql/AsDataFetcherResult.kt

```diff
@@ -0,0 +1,23 @@
+package suwayomi.tachidesk.graphql
+
+import com.expediagroup.graphql.server.extensions.toGraphQLError
+import graphql.execution.DataFetcherResult
+import mu.KotlinLogging
+
+inline fun <T> asDataFetcherResult(block: () -> T): DataFetcherResult<T?> {
+    val result =
+        runCatching {
+            block()
+        }
+
+    if (result.isFailure) {
+        KotlinLogging.logger { }.error(result.exceptionOrNull()) { "asDataFetcherResult: failed due to" }
```

Comment:
Bring KotlinLogging.logger {} outside the function, we don't need to recreate the logger every time

Reply:
done